contents/handbook/growth/customer-support.md

```diff
@@ -16,7 +16,7 @@ We have a few different routes for users to contact us. As an open source compan
 
 These are the ways in which customers can currently reach us, in order of popularity:
 
-- **Slack** - our active [PostHog Users Slack](https://posthog.com/slack) community frequently post questions
+- **Slack** - our active [PostHog Users Slack](https://posthog.com/slack) community frequently post questions, and for [Scale](/pricing#scale) and [Scale](/pricing#enterprise) customers, through a dedicated channel on our main company Slack
```

Comment:
We don't officially support scale customers with private slack channels, though we do it in practice and there are some legacy customers as well. Probably fine to leave in handbook, but this should read [Scale](link) and [Enterprise](link). Otherwise looks good to me!